### ISSUE TITLE
Migrate existing scripts from Shell to Python

### ISSUE THREAD

**Issue by maxachis** [Police-Data-Accessibility-Project/prod-to-dev-migration#7]
The first version of this repository was intended to be fairly lightweight, and hence it was supposed that shell scripts would be sufficient.

As the repository has expanded in complexity, now leveraging separate Stage and Sandbox logic, and the possible insertion of dummy data, shell scripts become decreasingly justifiable. Shell code is generally less flexible, less clear, and less familiar to the majority of developers. Future maintenance and development of this data may be hampered by reliance on shell code in every component of logic.

Thus, it will likely be useful in the long run to convert the code from Shell to Python. Python's error handling is more robust, the code is generally more clear, and it is the language most developers will be familiar with it. Unclear whether all shell scripts should be migrated to Python, but at least some of them likely should be.

**Comment by ghost** [comment]
I'd like to take a stab at this. I'm new to the whole "contributing to open source" thing, though. Can I contact someone to give me some guidelines/assistance?

**Comment by maxachis** [comment]
> I'd like to take a stab at this. I'm new to the whole "contributing to open source" thing, though. Can I contact someone to give me some guidelines/assistance?

Certainly! A good first step would be to direct you to our discord server, where you'll be able to reach out to folks and get acquainted: https://discord.gg/Er8UPvGJ

I will say that this issue is mostly taken care of (and I should update the issue to reflect that). However, there are a _wide_ number of issues available in some of our other repositories, most notably [this one](https://github.com/Police-Data-Accessibility-Project/data-sources-app).
